doc/clustering.md

```diff
@@ -93,7 +93,7 @@ Those typically cover:
  - Source device for a storage pool (leaving empty would create a loop)
  - Name for a ZFS zpool (defaults to the name of the LXD pool)
  - External interfaces for a bridged network (empty would add none)
- - Name of the parent network device for managed physical or macvlan networks (must be set)
+ - Name of the parent network device for managed physical or Macvlan networks (must be set)
```

Comment:
I'm wondering why "M" is upper cased here but "P" in "physical" isn't, or rather why was it not OK with "m" being lower cased?

Reply:
It was a decision I made when I worked on the networking section - there's no predominant capitalization when googling, so I just went for one option. I'm fine to change to either "macvlan" or "MacVLAN" though.

Reply:
I've got no issue with "Macvlan" itself, but the inconsistency of how we use it compare to the "physical" network reference here.

In this context both are references to LXD network type names, and we use upper-case-first for all LXD network type names elsewhere (such as https://linuxcontainers.org/lxd/docs/master/howto/network_create/).

So it should be:

> Name of the parent network device for managed Physical or Macvlan networks (must be set)

to be consistent showing that we using LXD network type names here.

Reply:
> In this context both are references to LXD network type names, and we use upper-case-first for all LXD network type names elsewhere (such as https://linuxcontainers.org/lxd/docs/master/howto/network_create/).

No, we don't - the table just uses the page titles. We're talking about "a bridge network", "an OVN network", "a Macvlan network", "a(n?) SR-IOV network" and "a physical network". The reason for that is that "bridge" and "physical" are actual words that describe the network (a network that uses a network bridge, a network that uses a physical connection), while the others are network names. 
Now, since there is no "official" source for Macvlan, I'm not sure if it really should be treated as a proper name, so I'm totally fine with talking about "a macvlan network" as well. But then there is still a difference between "a macvlan network" and "a network of type `macvlan`" (just like an OVN network has network type `ovn`).

Reply:
I'm not sure I agree. Here we aren't referring to an actual physical network (the thing) but a named entity type that is the LXD managed Physical network. So in this case it is isn't just any "bridge network" but in fact a LXD managed Bridge network. This is actually an important to differentiate because we can have `bridged` NIC types that connect to an unmanaged (i.e any) bridge networks using the `parent` setting but when using the `network` setting they can only connect to a managed Bridge network (that is a specific LXD entity type name).
 
So we can have "bridge networks" and "managed Bridge networks" and they are different. The former is just a bridge interface on a host and the latter is that plus a dnsmasq service, firewall and NAT rules. 

If it makes it easier we can just use the `bridge` reference anywhere we are referring to the managed Bridge network type as that would help identify what we are talking about here (perhaps there is something you can add that makes it link to the `bridge` network documentation where we are referring to a managed bridge network)?

Reply:
But to bring it back to the case in point "physical". There can be "physical networks" and "LXD Physical networks". The former is just an interface on a host connected to some external network. The latter is a named entity type that must be created that contains configuration as well as a reference to a named physical interface. In this case we are referring to the latter named entity type rather than just any "physical network".

Reply:
We're pretty clear though (at least in the cleaned up parts of the docs ;) ) on when we're talking about managed networks, and most of the time we say "network of type `xxx`" anyway. 
I'm absolutely against capitalizing everything that could be regarded as a proper name for a concept, because then we'll be capitalizing every other word in the end. ;) 

Let's just change "Macvlan" to "macvlan" and make sure that we are clear about when we are talking about managed networks and when not. And whenever there's any doubt, we'll mention the network type instead of a name - that's more precise anyway.

Reply:
OK thanks, yes mentioning the network name like `macvlan` or `bridge` would be a good way to indicate it is a LXD entity rather than an external subject. And it avoids the capitlisation debate :)